### PR TITLE
Bug fixes for the Twilio Connector User Doc

### PR DESCRIPTION
**Pull request by dhirsch1**

### CODE REVIEW

File: modules/ROOT/pages/twilio/twilio-connector.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= Twilio Connector
+= Twilio Connector - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
```
Comment (kbostrom) [inline modules/ROOT/pages/twilio/twilio-connector.adoc:4]: HIDDEN TEXT
Minor edits; please compare to orig.
Is there no text between the title and this list?

* Get Message List
* Redact Message
* Send Message
These fields can accompany an operation:
[%header,cols="30s,70a"]
|===
|Field |Description
|Account Sid |The unique ID of the account that sent this message
|Date Created |When the resource was created
|Media Sid |Unique ID for the media
|Message Sid |Unique ID for the message, generated after you send a message
|Date Sent |Date the message was sent from Twilio, in RFC 2822 format
|From |Phone number or sender ID that initiated a message, in E.164 format.
|To |Phone number that received a message, in E.164 format
|Entity Reference |MEL expression for what you want to view, such as the payload
|===
For more information about each format, see the See Also section at the end of this document.

File: modules/ROOT/pages/twilio/twilio-connector.adoc [hunk 2]
```diff
@@ -148,9 +148,9 @@ For more information on each format, see the See Also section at the end of this
...
 == Use Case: Studio
 
-In the following example, a Mule application sends a message to a phone number, and then redacts it.
+In the following example, a Mule application sends a message to a phone number, and then redacts it. This example uses variables for field values. You can provide the values for each variable in a properties file. To use a properties file, place the file at `/src/main/resources` and use the tag `<configuration-properties>` to point to the file. For more information, see xref:mule-runtime::configuring-properties.adoc#properties-files[Properties Files].
 
-. Create a new Mule application and add the following properties to the mule-app.properties file:
+. Create a new Mule application and add the following properties to the properties file:
 +
 [%header,cols="30s,70a"]
 |===
```
Comment (kbostrom) [inline modules/ROOT/pages/twilio/twilio-connector.adoc:158]: HIDDEN TEXT
Edited; please compare
(mostly for lowercase and no periods....)

|Property |Description
|accountSid |Your account SID
|authToken |Your authentication token
|fromNumber |The phone number from which the SMS is to be sent (the phone number you received from Twilio)
|===
+
. Add an empty flow, drag an HTTP connector to the inbound part of the flow, and set its path to `/send/{toNumber}`.
. Drag a transform message to the flow and prepare the input for the Twilio connector:
+
[source,dataweave,linenums]
----
%dw 2.0
output application/java
---

File: modules/ROOT/pages/twilio/twilio-connector.adoc [hunk 3]
```diff
@@ -172,7 +172,7 @@ output application/java
 {
     Body: "You are now subscribed!",
     From: "${fromNumber}",
-    To: "+" ++ inboundProperties.'http.uri.params'.toNumber
+    To: "+" ++ attributes.uriParams.toNumber default "my default value"
 }
 ----
 +
```
Comment (kbostrom) [inline modules/ROOT/pages/twilio/twilio-connector.adoc:179]: HIDDEN TEXT
Edited, please compare

. Add a Twilio Connector after the transform message and apply the following settings:
** Select the Send Message operation.
** Set Account Sid to `${accountSid}`, and Entity Reference to `#[payload]`.
+
. Drag a variable component and configure the following parameters:
** Set Name to `messageSid`.
** Set Value to `#[payload.getSid()]`.
+
. Add another transform message to create the input for the Redact Message operation:
+
[source,dataweave,linenums]
----
%dw 2.0
output application/java
---
{
    Body: "",
    From: payload.from,
    To: payload.'to'
}

File: modules/ROOT/pages/twilio/twilio-connector.adoc [hunk 4]
```diff
@@ -148,9 +148,9 @@ For more information on each format, see the See Also section at the end of this
 
 == Use Case: Studio
 
-In the following example, a Mule application sends a message to a phone number, and then redacts it.
+In the following example, a Mule application sends a message to a phone number, and then redacts it. This example uses variables for field values. You can provide the values for each variable in a properties file. To use a properties file, place the file at `/src/main/resources` and use the tag `<configuration-properties>` to point to the file. For more information, see xref:mule-runtime::configuring-properties.adoc#properties-files[Properties Files].
```
Comment (kbostrom) [inline modules/ROOT/pages/twilio/twilio-connector.adoc]:
```suggestion
In the following example, a Mule application sends a message to a phone number, and then redacts it. 

Note that although this example uses variables for field values, you can provide actual values for each variable by using a properties file. To use a properties file, place the file at `/src/main/resources` and use the tag `<configuration-properties>` to point to the file. For more information, see xref:mule-runtime::configuring-properties.adoc#properties-files[Properties Files].
```
Could the sentence "To use a properties file..." be omitted, given the cross-ref.?

File: modules/ROOT/pages/twilio/twilio-connector.adoc [hunk 5]
```diff
@@ -148,9 +148,9 @@ For more information on each format, see the See Also section at the end of this
...
 
-. Create a new Mule application and add the following properties to the mule-app.properties file:
+. Create a new Mule application and add the following properties to the properties file:
```
Comment (kbostrom) [inline modules/ROOT/pages/twilio/twilio-connector.adoc:155]: I'm confused: the example also uses a properties file?
Maybe we don't need the long explanation above, if the example already assumes that the audience knows how to use a properties file. Also, aren't most audiences savvy enough to recognize variable values?

**Review by kbostrom — APPROVED** [review]
Done

File: modules/ROOT/pages/twilio/twilio-connector.adoc (hunk 3 above)
Comment (dhirsch1) [reply modules/ROOT/pages/twilio/twilio-connector.adoc:179]: FYI, Transform message is the name of the component. I put "Transform message component." I changed "Variable" to "Set Variable component," also as shown in Studio.

File: modules/ROOT/pages/twilio/twilio-connector.adoc (hunk 5 above)
Comment (dhirsch1) [reply modules/ROOT/pages/twilio/twilio-connector.adoc:155]: Part of the bug was that we should be explaining where the properties file is located, so I am leaving this in.

File: modules/ROOT/pages/twilio/twilio-connector.adoc (hunk 4 above)
Comment (dhirsch1) [reply modules/ROOT/pages/twilio/twilio-connector.adoc]: Yes to eliminating "To use a properties file." No to "although." It changes the technical meaning and is incorrect.

**Pull request merged**